frontend/scss/components/templates/component-detail.scss

```diff
@@ -33,6 +33,8 @@ body {
   .#{utility('help')} {
     @include content;
     @include content-white-bg;
+    padding-left: 0px;
```

Comment:
Omit `px` as zero is zero no matter the unit.

Reply:
corrected